### PR TITLE
Create timer.py

### PR DESCRIPTION
**Pull request by dandre2**
on line 31 it says timeleft not defined.

### CODE REVIEW

**Comment by rakirs2** [comment]
Ahh, so this is an AMAZING problem. One of my favorites.

Highest to lowest

1. Remember how the computer reads the code.
2. What does the computer know at the time it is reading that code
3. What is the error you're getting/are there examples of it.

So, let's look at what you have and a bit of code that is slightly different, but doesn't have the same problem.

Your code:
```
timeLabel = tkinter.Label( text = "Time left: " +
                (timeleft), font = ('Helvetica', 12))
```

Slighty different:
```
global timeleft;
timeleft = 10

timeLabel = tkinter.Label( text = "Time left: " +
                (timeleft), font = ('Helvetica', 12))
```

Basically, python needs to know that time left is instantiated. These types of problems are one of many reasons why globals aren't loved by most engineers.
